### PR TITLE
PG-239: Made LanguageIso return a default if language is not specified.

### PR DESCRIPTION
**Pull request by tombogle**

### CODE REVIEW

File: SIL.DblBundle/Bundle.cs [hunk 1]
```diff
@@ -96,9 +103,17 @@ protected Bundle(string pathToZippedBundle)
 		public string Id { get { return m_dblMetadata.Id; } }
 
 		/// <summary>
-		/// 3-letter ISO 639-2 code for the language of the DBL bundle
+		/// 3-letter ISO 639-2 code for the language of the DBL bundle. If the metadata's language does
+		/// not identify an ISO 639-2 code, this returns "qaa", which is the code indicating "unknown".
```
Comment (andrew-polk) [inline SIL.DblBundle/Bundle.cs]: After your change, this no longer returns qaa by default.

File: SIL.DblBundle/Bundle.cs [hunk 2]
```diff
@@ -81,6 +81,13 @@ protected Bundle(string pathToZippedBundle)
 
 		#region Public properties
 		/// <summary>
+		/// Typically, this should be set to SIL.WritingSystems.WellKnownSubtags.UnlistedLanguage by the client.
+		/// We would have probably just hard-coded to that, but we didn't want to have to refernce SIL.WritingSystems
```
Comment (andrew-polk) [inline SIL.DblBundle/Bundle.cs]: refernce -> reference

**Comment by darcywong00** [comment]
Ignore the noise from Bob.  It is building using the wrong dependencies in this case (its expecting the PR to be against master in effect).

**Comment by darcywong00** [comment]
Added the redundant dependency on the TC master build config so now it handles PRs on the SILWritingSystems branch ok.

**Pull request merged**